spec/support/bundle_setup.rb

```diff
@@ -16,3 +20,22 @@ def context_from_proj(proj)
 def noko_doc(x)
   Nokogiri.XML(x) { |conf| conf.default_xml.noblanks }
 end
+
+
+def context_from_proj_ar_model(proj)
```

Comment:
also not loving this method name.  bundle_context_from_hash ?  should this even be split out as a separate method?

Reply:
#275